hash_commands.go

```diff
@@ -15,7 +15,7 @@ type HashCmdable interface {
 	HSet(ctx context.Context, key string, values ...interface{}) *IntCmd
 	HMSet(ctx context.Context, key string, values ...interface{}) *BoolCmd
 	HSetNX(ctx context.Context, key, field string, value interface{}) *BoolCmd
-	HScan(ctx context.Context, key string, cursor uint64, match string, count int64) *ScanCmd
+	HScan(ctx context.Context, key string, cursor uint64, match string, count int64, noValues bool) *ScanCmd
```

Comment:
To me this is a breaking change. I opened this PR mostly to get the discussion going, curious how this is handled.